tests/webpack-localize-assets-plugin.spec.ts

```diff
@@ -462,6 +536,47 @@ describe(`Webpack ${webpack.version}`, () => {
 			expect(mfs.readFileSync('/dist/index.ja.js').toString()).not.toMatch(/\s{2,}/);
 		});
 
+		test('custom localisation compiler works with minification', async () => {
+			const buildStats = await build(
+				{
+					'/src/index.js': 'export default __("hello-key") + " world and " + __("stringWithQuotes");',
+				},
+				(config) => {
+					config.optimization!.minimize = true;
+					config.plugins!.push(
+						new WebpackLocalizeAssetsPlugin({
+							locales: localesMulti,
+							localizeCompiler(callArguments, localeName) {
+								return `'${localeName}-${callArguments[0].slice(1, -1)}'`;
```

Comment:
Maybe you're trying to test minification here but this will break depending on the minifier since this yields invalid JS:
```js
const o='en-hello-key'world and'en-stringWithQuotes'
```

----

If we're applying localizeCompiler post-minification, we can remove this test entirely.

Reply:
> this will break depending on the minifier

That’s why I wanted to write a test for it!

Reply:
My point is, Terser (Webpack's default minifier) should be throwing a parsing error for this, and since there are many different minifiers a Webpack user can use, we don't know which ones this will break on.

Adding this test only verifies that Terser is lenient on invalid code, which seems out of scope for this plugin.

Reply:
Not quite sure I follow you. The plugin generates `export default 'en-hello-key' + " world and " + 'en-stringWithQuotes';` ([see test](https://github.com/privatenumber/webpack-localize-assets-plugin/pull/21/files#diff-6a0a8bbfc6a49ad17e94f931f95c6123f91d3476c9b1dbd41bb45a511a43b36aR566)). It only generates incorrect code when running post-minification, which is what this test is supposed to catch.

Reply:
Oh you're right -- my mistake. I must've been playing with the code and had the pre-minification transformations disabled and forgot to revert it.

Anyway, if we can prevent the minifier from interfering with the placeholders, I think this test can be removed. 👍

Reply:
Agreed, if I implement your idea (https://github.com/privatenumber/webpack-localize-assets-plugin/pull/21#discussion_r744294701) I think the behaviour will be covered by the existing test.

Reply:
https://github.com/privatenumber/webpack-localize-assets-plugin/pull/21/commits/2626e5ce875a5671f2a4ffd27ac652e1b9cda44f